contracts/subdaos/cwd-subdao-timelock-single/src/state.rs

```diff
@@ -6,3 +6,5 @@ pub const DEFAULT_LIMIT: u64 = 30;
 
 pub const CONFIG: Item<Config> = Item::new("config");
 pub const PROPOSALS: Map<u64, SingleChoiceProposal> = Map::new("proposals");
+/// Execution errors for proposals that do execute only once
+pub const PROPOSAL_FAILED_EXECUTION_ERRORS: Map<u64, String> = Map::new("failed_proposal_errors");
```

Comment:
why key is `failed_proposal_errors`, not exactly as the variable name? that would be nice to keep these two things consistent